graphql_api/types/repository/repository.graphql

```diff
@@ -84,6 +84,7 @@ type Repository {
   componentsYaml(termId: String): [ComponentsYaml]!
   testAnalyticsEnabled: Boolean
   isGithubRateLimited: Boolean
+  # TODO - remove with #2291
   testResults(
```

Comment:
`testResults` is being used by frontend client, so will remain here until frontend cut over, then can be removed